pkg/wallet/api.go

```diff
@@ -20,23 +20,14 @@ type Wallet interface {
 // Crypto interface
 type Crypto interface {
 
-	// CreateEncryptionKey create a new public/private encryption keypair.
+	// CreateKeyPairSet create a new public/private encryption and signagure keypairs combo.
 	//
 	// Returns:
 	//
-	// string: verKey
-	//
-	// error: error
-	CreateEncryptionKey() (string, error)
-
-	// CreateSigningKey create a new public/private signing keypair.
-	//
-	// Returns:
-	//
-	// string: verKey
-	//
+	// string: enc public key of the encryption keypair
+	// string: sig public key of the signature keypair
 	// error: error
-	CreateSigningKey() (string, error)
+	CreateKeyPairSet() (string, string, error)
```

Comment:
I also find it confusing to return two keys, especially when you look at usages of this method. It is ok for now since it will be fixed in subsequent PR.

Reply:
Agreed.

When we switch to the new keys structures (ref #596), this function will return only 1 reference value from which sub keys can be fetched later with only this id.

For now there is only 1 wallet storage and both signing and encryption keys are stored in 1 document, this will change to include a metadata storage to reference keys relationships.